config/locales/activerecord.yml

```diff
@@ -15,6 +30,14 @@ en:
       year_invalid: must include a 4-digit year
       email:
         invalid: is not a valid email address
+  activemodel:
+    errors:
+      models:
+        job_specification_form:
+          attributes:
+            <<: *vacancy_errors_labels
```

Comment:
what does the `<<` do here? Just curious about it 😄

Reply:
"its a yaml thing dushan!" - Adrian

Reply:
https://stackoverflow.com/questions/6651275/what-do-the-mean-in-this-database-yml-file

Reply:
Thank you!

Reply:
@dushan-madetech In this case, `<<` adds all the keys under the defined `vacancy_errors_labels` to the attributes under `job_specification_form`. It reduces having to rewrite the same keys over and over again which would take up lines.